### PR TITLE
EHUB-315 Simplify and clarify the GitHub Actions CI tutorial.

### PR DESCRIPTION
**Pull request by nshaw**

### CODE REVIEW

File: vuepress/docs/next/tutorials/ecr/github-actions-workflow.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+
+## Create a backend workflow
+
+1. Go to your main project folder in the shell
+2. Run the JHipster `ci-cd` subgenerator
+```bash
+ent jhipster ci-cd
+```   
+2. Select `GitHub Actions`
+3. Click ENTER to choose no `tasks/integrations` and generate the default workflow
```
Comment (Lyd1aCla1r3) [inline vuepress/docs/next/tutorials/ecr/github-actions-workflow.md:23]: Workflow capitalized? (Comment also applies to all instances below...) Anthony and I had agreed on Workflow due to the fact that references were to a specific workflow, and Workflow is capitalized in the GitHub documentation... it's a gray area.

File: vuepress/docs/next/tutorials/ecr/github-actions-workflow.md [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+5. Commit the workflow file and push it to github.
+```bash
+git add .github
+git commit -m "Add the backend CI job"
+git push
+```
+6. Since the workflow runs on ```push``` or ```pull-request```, you can immediately check the GitHub ```Repository → Actions``` tab to see the status of the corresponding Jobs. The standard tests in a Blueprint-generated project can take a few minutes to run.
+* You can review the logs for each step to diagnose CI failures
+* If the workflow fails or times out, you should receive an email notification
+* You can choose to skip the CI workflow by including `skip ci` or `ci skip` in your commit message. This can be customized in the job definition.
```
Comment (Lyd1aCla1r3) [inline vuepress/docs/next/tutorials/ecr/github-actions-workflow.md:34]: same as above wrt Job or job...

File: vuepress/docs/next/tutorials/ecr/github-actions-workflow.md [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+3. Click ENTER to choose no `tasks/integrations` and generate the default workflow
+4. The initial workflow file is available at `.github/workflows/github-ci.yml`
+5. Commit the workflow file and push it to github.
+```bash
+git add .github
+git commit -m "Add the backend CI job"
+git push
+```
+6. Since the workflow runs on ```push``` or ```pull-request```, you can immediately check the GitHub ```Repository → Actions``` tab to see the status of the corresponding Jobs. The standard tests in a Blueprint-generated project can take a few minutes to run.
+* You can review the logs for each step to diagnose CI failures
```
Comment (Lyd1aCla1r3) [inline vuepress/docs/next/tutorials/ecr/github-actions-workflow.md:32]: would add period for consistency

File: vuepress/docs/next/tutorials/ecr/github-actions-workflow.md [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+* If the workflow fails or times out, you should receive an email notification
```
Comment (Lyd1aCla1r3) [inline vuepress/docs/next/tutorials/ecr/github-actions-workflow.md:33]: would add period for consistency

File: vuepress/docs/next/tutorials/ecr/github-actions-workflow.md [hunk 5]
```diff
@@ -0,0 +1,107 @@
...
+* You can choose to skip the CI workflow by including `skip ci` or `ci skip` in your commit message. This can be customized in the job definition.
+* See the [GitHub Actions](https://github.com/features/actions) documentation for more information on alternative trigger options.
+
+Next let's expand the workflow to also include the micro frontends.
```
Comment (Lyd1aCla1r3) [inline vuepress/docs/next/tutorials/ecr/github-actions-workflow.md:37]: add comma - "Next, let's.."

File: vuepress/docs/next/tutorials/ecr/github-actions-workflow.md [hunk 6]
```diff
@@ -0,0 +1,107 @@
...
+
+## Add a frontend job to the workflow
+We'll now add a second job to the workflow definition specifically for the micro frontends. Optionally, you could also create a completely separate workflow file if you'd prefer.
```
Comment (Lyd1aCla1r3) [inline vuepress/docs/next/tutorials/ecr/github-actions-workflow.md:40]: add comma: "definition, specifically" @jyunmitch check me on this :P

File: vuepress/docs/next/tutorials/ecr/github-actions-workflow.md [hunk 7]
```diff
@@ -0,0 +1,107 @@
...
+           npm test
+```
+2. (Optional) If your widget path differs from the standard Blueprint example, you should change the first command in `Run tests` to match your path.
+3. Commit and push the changes to GitHub.
+```   
+   git add .github
+   git commit -m "Add the microfrontend CI job"
+   git push
+```
+4. Now check the GitHub ```Repository → Actions``` tab to see the status of the Jobs. You should see something like this when the `micro-frontends` job is complete:
```
Comment (Lyd1aCla1r3) [inline vuepress/docs/next/tutorials/ecr/github-actions-workflow.md:66]: to immediately introduce the reference example I'd rephrase: "When the `micro-frontends` Job is complete, you should see something like this:"

File: vuepress/docs/next/tutorials/ecr/github-actions-workflow.md [hunk 8]
```diff
@@ -0,0 +1,107 @@
...
+```
+Test Suites: 2 passed, 2 total
+Tests:       4 passed, 4 total
+Snapshots:   0 total
+Time:        2.911s
+Ran all test suites.
+```
+
+## Extend the frontend job to cover multiple MFEs
+You may have multiple micro frontends or widgets in your project. One option is to duplicate the job for each MFE but you can also use the GitHub Actions matrix feature to avoid duplicating those definitions.
```
Comment (Lyd1aCla1r3) [inline vuepress/docs/next/tutorials/ecr/github-actions-workflow.md:76]: Project vs project
Should introduce MFE as acronym above the first time micro frontend is referenced
add comma: "...each MFE, but..."

File: vuepress/docs/next/tutorials/ecr/github-actions-workflow.md [hunk 9]
```diff
@@ -0,0 +1,107 @@
...
+
+1. Change your job definition to the following. Note the following changes we've made here:
```
Comment (Lyd1aCla1r3) [inline vuepress/docs/next/tutorials/ecr/github-actions-workflow.md:78]: would remove second "following": "Note the changes..."

File: vuepress/docs/next/tutorials/ecr/github-actions-workflow.md [hunk 10]
```diff
@@ -0,0 +1,107 @@
...
+* The `job.name` is parametrized based on the MFE matrix name
+* The `job.strategy` has been set to `fail-fast:false` so all MFEs will be tested rather than stopping the job on the first failure
```
Comment (Lyd1aCla1r3) [inline vuepress/docs/next/tutorials/ecr/github-actions-workflow.md:80]: add comma: "...tested, rather..." ?? @jyunmitch

**Review by Lyd1aCla1r3 — COMMENTED** [review]
reads really well and very clear!

File: vuepress/docs/next/tutorials/ecr/github-actions-workflow.md (hunk 1 above)
Comment (nshaw) [reply vuepress/docs/next/tutorials/ecr/github-actions-workflow.md:23]: I think uppercase when referring to the feature but lowercase when referring to a specific workflow. That seems to be how GitHub does it.
image

File: vuepress/docs/next/tutorials/ecr/github-actions-workflow.md (hunk 2 above)
Comment (nshaw) [reply vuepress/docs/next/tutorials/ecr/github-actions-workflow.md:34]: Same, lowercase job, I think. It's lowercase in the yaml so probably good to match it here.

File: vuepress/docs/next/tutorials/ecr/github-actions-workflow.md (hunk 3 above)
Comment (nshaw) [reply vuepress/docs/next/tutorials/ecr/github-actions-workflow.md:32]: Got it.

File: vuepress/docs/next/tutorials/ecr/github-actions-workflow.md (hunk 5 above)
Comment (nshaw) [reply vuepress/docs/next/tutorials/ecr/github-actions-workflow.md:37]: Got it.

File: vuepress/docs/next/tutorials/ecr/github-actions-workflow.md (hunk 4 above)
Comment (nshaw) [reply vuepress/docs/next/tutorials/ecr/github-actions-workflow.md:33]: Got it.

File: vuepress/docs/next/tutorials/ecr/github-actions-workflow.md (hunk 7 above)
Comment (nshaw) [reply vuepress/docs/next/tutorials/ecr/github-actions-workflow.md:66]: Got it

File: vuepress/docs/next/tutorials/ecr/github-actions-workflow.md (hunk 8 above)
Comment (nshaw) [reply vuepress/docs/next/tutorials/ecr/github-actions-workflow.md:76]: Done

**Review by Lyd1aCla1r3 — APPROVED** [review]
